packages/x-charts/src/hooks/usePieSeries.ts

```diff
@@ -43,3 +47,16 @@ export function usePieSeries(seriesIds?: SeriesId | SeriesId[]) {
 export function usePieSeriesContext(): UsePieSeriesContextReturnValue {
   return useAllSeriesOfType('pie');
 }
+
+/**
+ * Get access to the pie layout.
+ * @returns {Record<SeriesId, PieSeriesLayout>} the pie layout
+ */
+export function usePieSeriesLayout(): Record<SeriesId, PieSeriesLayout> {
```

Comment:
If it is not meant to be external it is probably better to move this to a plugin or to where it is used 🤔 

Else we should probably export it from /hooks

Reply:
Yeah but waiting for a headless version of the pie could help having a clear DX instead of an additional hook to document ;)